velox/dwio/dwrf/test/ReaderTest.cpp

```diff
@@ -253,14 +263,15 @@ void verifyPrefetch(
 //   memo varchar
 // }
 void verifyFlatMapReading(
+    memory::MemoryPool& pool,
```

Comment:
usually memory pool is passed as a raw pointer

Reply:
Fixed